sources/packages/web/src/services/StudentAssessmentsService.ts

```diff
@@ -126,7 +126,7 @@ export class StudentAssessmentsService {
    */
   async getApplicationWarnings(
     applicationId: number,
-  ): Promise<ECertFailedValidation[]> {
+  ): Promise<ApplicationWarningsAPIOutDTO> {
```

Comment:
Minor. comment at line 125 can be updated.

Reply:
Changed the comment, slightly.